backoffice/src/Views/Email/Index.cshtml

```diff
@@ -1,7 +1,37 @@
+@model List<EmailView>
 @{
     ViewData["Title"] = "Emails";
 }
 
 <h4 class="display-8">@ViewData["Title"]</h4>
 
+<table class="table">
+    <thead>
+    <tr>
+        <th scope="col">Recipient</th>
+        <th scope="col">Subject</th>
+        <th scope="col">Type</th>
+        <th scope="col">Last Modified</th>
+        <th scope="col">Actions</th>
+    </tr>
+    </thead>
+    <tbody>
+    @foreach (var email in Model)
+    {
+        <tr>
+            <th scope="row">
+                <a asp-route-id="@email.Id">@email.Recipient</a>
```

Comment:
probably wanna have asp-action too "View"

Reply:
currently don't have a view for emails only a Preview view.

Reply:
Yeah that's fine, "Preview". Currently this will try to go to /Email/${Email.Id}

If no controller is specified, it assumes the same controller, not sure what it would do with just an Id